docker/exec_tests.sh

```diff
@@ -0,0 +1 @@
+docker-compose exec truffle yarn run test:ci
```

Comment:
Probably should add the usual shebang etc.

Remember to `chmod +x`. Can't tell.

Reply:
I deleted the script, seemed useless anyway.